domain/main.go

```diff
@@ -211,10 +211,10 @@ func (e *employeeService) UpdateEmployee(ctx context.Context, req *proto.Employe
 
 func (e *employeeService) AddVacation(ctx context.Context, req *proto.VacationRequest) (*proto.Vacation, error) {
 	start := time.Unix(req.StartDate, 0)
-	tomorrow := time.Now().Add(24 * time.Hour)
+	hours24 := time.Now().Add(24 * time.Hour)
 
-	afterTomorrow := start.After(tomorrow)
-	if !afterTomorrow {
+	after24Hours := start.After(hours24)
+	if !after24Hours {
```

Comment:
you can put both lines in a single if statement, as you dont need `after24Hours` later in the function